build.gradle

```diff
@@ -195,7 +195,8 @@ def sonarExclusions = [
   '**/uk/gov/hmcts/reform/hmc/api/exceptions/**',
   '**/uk/gov/hmcts/reform/hmc/api/enums/**',
   '**/uk/gov/hmcts/reform/hmc/api/services/**',
-  '**/uk/gov/hmcts/reform/hmc/api/controllers/HearingsController.java'
+  '**/uk/gov/hmcts/reform/hmc/api/controllers/HearingsController.java',
```

Comment:
Please remove the exclusion.

Reply:
Thank you @shashisk97 Removed exclusions